timeoutwrite.h

```diff
@@ -5,11 +5,11 @@
 
 extern ssize_t timeoutwrite(int t, int fd, const void *buf, size_t len);
 
-#define GEN_SAFE_TIMEOUTWRITE(funcname,tout,writefd,doexit) \
+#define GEN_SAFE_TIMEOUTWRITE(funcname,tout,doexit) \
 ssize_t funcname(int fd, const void *buf, size_t len) \
```

Comment:
The `fd` argument was never used there. Maybe an omission or a change of mind mid-way...
This sounds like pursuing some work, makes sense to me.
Thank you!

Reply:
Not entirely. Some callers passed "fd" as the "writefd" macro argument, so they actually used the fd passed to the function. And now just all do.